src/components/InputCheckbox/InputCheckbox.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { errorText, helpText } from '../../stories/helpers';
+
+import InputCheckbox from './InputCheckbox';
+import Fieldset from '../Fieldset';
+import { ChangeEvent } from 'react';
+
+const meta = {
+  title: 'Components/InputCheckbox',
+  component: InputCheckbox,
+  parameters: {
+    layout: 'centered'
+  },
+  tags: ['autodocs'],
+  args: {
+    label: 'Checkbox'
+  }
+} satisfies Meta<typeof InputCheckbox>;
+
+export default meta;
+type Story = StoryObj<typeof InputCheckbox>;
+
+export const Playground: Story = {};
+
+export const WithHelp: Story = {
+  name: 'With help',
+  args: {
+    helpText
+  }
+};
+
+export const WithError: Story = {
+  name: 'With error',
+  args: {
+    errorText,
+    forceValidation: true
+  }
+};
+
+export const WithHelpAndError: Story = {
+  name: 'With help and error',
+  args: {
+    helpText,
+    errorText,
+    forceValidation: true
+  }
+};
+
+let checkboxes = [
+  {
+    text: 'Checkbox 1',
+    isChecked: true
+  },
+  {
+    text: 'Checkbox 2',
+    isChecked: false
+  },
+  {
+    text: 'Checkbox 3',
+    isChecked: false
+  }
+];
+
+const handleChange = (e: ChangeEvent<HTMLInputElement>, index: number) => {
+  const updatedCheckboxes = [...checkboxes];
+  updatedCheckboxes[index].isChecked = e.target.checked;
+  checkboxes = updatedCheckboxes;
+};
+
+export const Group: Story = {
```

Comment:
thoughts on moving group into fieldset? we may want to add a TODO to talk to catie when she's back since i could see this either way since radio group is with radio, but those can never be used apart